src/Tests/Blockcore.IntegrationTests/Blockcore.IntegrationTests.csproj

```diff
@@ -70,6 +70,9 @@
     <None Update="Wallet\Data\test.wallet.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="Wallet\Data\wallet-with-funds.txdb.litedb">
```

Comment:
Comparing this to the old format, ".wallet.json", would not not make more sense to have "wallet.db"? Perhaps we should reserve wallet.db for a future LiteDB implementation of the wallet addresses themselves?

Reply:
hmm no we dont want to mention wallet in this file (thats why the folder is called txdb to avoid feeling those files are important to backup)